website/docs/guides/fusion-upgrade-prepare.md

```diff
@@ -0,0 +1,459 @@
+---
+title: "Upgrade to Fusion part 1: Preparing to upgrade"
+id: "prepare-fusion-upgrade"
+level: 'Intermediate'
+icon: 'zap'
+hide_table_of_contents: true
```

Comment:
I wonder if we should set to `false` /show ToC because there are so many subsections. It might be nice to see what you're getting into.